lib/actions-oclif/os/configure.ts

```diff
@@ -308,6 +308,8 @@ async function validateOptions(options: FlagsDef) {
 			-------------------------------------------------------------------------------------------
 		`);
 	}
+
+	// TODO: This should be converted to this.checkLoggedIn, if this function is moved into OsConfigureCmd class.
 	const { checkLoggedIn } = await import('../../utils/patterns');
```

Comment:
You could make the `checkLoggedIn` function static, and it means if the "backend" of the function ever changes, even this invocation would change.

Reply:
@CameronDiver Yes right.  The reason `checkLoggedIn` wasn't static is that it was originally checking some values in the subclass, that would not have been accessible (these checks now happen in init() now though).

Reply:
Though we'd have to make `checkLoggedIn` `public`, but not a big deal.